src/user_stats/stats.rs

```diff
@@ -0,0 +1,80 @@
+use super::*;
+
+/// Achievement API.
+///
+/// Methods require
+/// [`request_current_stats()`](../struct.UserStats.html#method.request_current_stats)
+/// to have been called and a successful [`UserStatsReceived`](../struct.UserStatsReceived.html)
+/// callback processed.
+///
+/// # Example
+///
+/// ```no_run
+/// # use steamworks::*;
+/// # let (client, single) = steamworks::Client::init().unwrap();
+/// // Unlock the 'WIN_THE_GAME' achievement
+/// client.user_stats().achievement("WIN_THE_GAME").set()?;
+/// # Err(())
+/// ```
+pub struct AchievementHelper<'parent, M> {
```

Comment:
Unless something else is going to be called `Achievement` (I don't know, I haven't really looked into the achievement side of steamworks yet) this might as well be called `Achievement` instead.

Reply:
That would work too. I guess I called it helper as it doesn't directly map to the steam API interfaces, instead these are still in `ISteamUserStats`, it also is a reference based helper rather than an `Arc` one as the usage should be short-lived, builder style.

In any case I don't think it's a big deal either way, so let me know how you'd prefer it.

Reply:
Fair enough.